### ISSUE TITLE
.NET Core Bindings: Bindings not discovered on slow machines

### ISSUE THREAD

**Issue by gasparnagy** [specsolutions/deveroom-feedback#11]
When the solution is opened in a way that a feature file gets active immediately (feature file was open and active before solution close), on some slow machines the bindings are not discovered. 

On the output pane it displays:
```
Info: .ctor: Starting Deveroom...
Info: CreateProjectScope: Initializing project: BddWithSpecFlow.GeekPizza.Specs
Info: StartRetryInitializeTimer: Invalid project settings collected, retry in 5 seconds...
Info: StartRetryInitializeTimer: Invalid project settings collected, retry in 5 seconds...
Info: StartRetryInitializeTimer: Invalid project settings collected, retry in 5 seconds...
Info: RetryInitializeTimerTick: Project settings could not be initialized. Build the project to reload settings.
```

The bug is that the bindings are not discovered even if you rebuild afterwards.

Workaround: close all feature files, close and reopen solution, build the project and only after open the first feature file.

**Comment by gasparnagy** [comment]
Better workaround: Rebuild helps

**Comment by gasparnagy** [comment]
fixed in v1.0.3
